frontend/src/routes/FacilityBooking/EditBooking/index.tsx

```diff
@@ -34,6 +34,26 @@ import SelectableField from '../../../components/SelectableField'
 import ButtonComponent from '../../../components/Button'
 import { get24Hourtime } from '../../../common/get24HourTime'
 
+/**
+ *
+ * @returns The page through which the user is able to edit his facility booking.
+ *
+ * @example
+ * ```
+ * The user can edit the following through this page:
+ *  1: Facility
+ *  2: Name of Event
+ *  3: Start/End Date/Time
+ *  4: CCA
+ *  5: Description
+ * ```
+ *
+ * @remarks
+ * <any remarks on this component type in here>
+ * Might make more sense to allow editing of the start/end date/time only and disable other fields.
```

Comment:
btw can remove the placeholder since you wrote stuff alr
```suggestion
 * Might make more sense to allow editing of the start/end date/time only and disable other fields.
```

Reply:
Hmm but regarding your remark, I believe there's still cases where ppl wanna change the booking name/description etc so idt it's an issue